editoast/src/client/redis_config.rs

```diff
@@ -12,3 +16,20 @@ pub struct RedisConfig {
     /// Redis url like `redis://[:PASSWORD@]HOST[:PORT][/DATABASE]`
     pub redis_url: String,
```

Comment:
```suggestion
    redis_url: String,
```
To force the use of `RedisConfig::url()`

Reply:
Done.